projects/packages/my-jetpack/_inc/components/product-card/index.jsx

```diff
@@ -38,13 +39,13 @@ const DownIcon = () => (
 
 const renderActionButton = ( {
 	status,
+	activateHandler,
```

Comment:
<img src="https://user-images.githubusercontent.com/77539/151169901-ed1b3c40-0474-41e8-b8b1-095caad7ecc3.png" width="600px" />

You're going to hate me, but removing the `on` from there makes it inconsistent in terms of naming. Usually, even in the core, all properties that expect a handler function starts with the on prefix.

Going a little further, the property name should start with `on` because it's considered as an event, while the function provided to handle the action can get a noun name, IMO.

```es6
<MyBlock
  onTurnLight={ turnTheLight }
>
```

Reply:
But I'm not removing the `on` from the component properties. The component still accepts `onActivate`, and `onDeactivate`. This is an internal function `renderActionButton` that renders a piece of the component.

Reply:
There's actually little value in this function existing IMO

Reply:
yeah, you're right but still :-). `renderActionButton()` has a couple of events such as `onLearn` ,`onAdd` ,`onManage` ,`onFixConnection` . Anyway, it's up to you. Feel free to ignore this comment. It's a minor suggestion.

Reply:
I think you're right. pushed dfd1e95ce38750d378a65e2740e5d8b7352dca13 that keeps onActivate